### DISCUSSION TITLE
MPB in North Dakota

### DISCUSSION THREAD

**Discussion by BartGnirk** [MyFirstBitcoin/.github#21] [LNN Introductions]
Hi everyone! I have started a Bitcoin Research Club at my college last year and want to teach MPB material at the local library starting this summer. The region is in the Midwest US located in Bismarck, ND and the language for the material will be English. I am looking forward to getting involved and learning more about the LNN!

**Comment by BitcoinSchoolAR** [comment]
Hi Bart!
Can you tell us more about what you've been up to with this Bitcoin Research Club? Sounds super fun! Greetings from Argentina.

**Reply by BartGnirk** [reply]
Of course, I started the club by going through the official process at the college. Had to find a full-time faculty professor to be the advisor and write a constitution. Once that was complete, flyers were posted around campus. This semester we have 3 solid members, hoping for some more to get involved after the summer break. Mostly meetings are open discussion about Bitcoin, inflation, monetary history, current events, educational resources, etc. I am working on a Bitcoin 101 presentation and would like to eventually do self-custody and node setup workshops. There are some challenges with being mostly an online student and making time around my work and family schedule. I think it's crucial to get to campus for events and club meetings. Layer 0 is the social layer of Bitcoin. 

- I recently filled out information to get involved with the Bitcoin Students Network https://www.bitcoinstudentsnetwork.org/
Hoping they can assist with ideas for growing interest in the club and resources also.
